src/components/NewCommentForm.tsx

```diff
@@ -14,24 +132,35 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': nameError,
+            })}
+            value={authorName}
+            onChange={event => {
```

Comment:
Pls move the function inside `onChange` handler to a separate const and use as a callback here
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
Still not fixed from the previous review 
<img width="847" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-posts/assets/64750958/8ae16cb5-4f94-407f-93eb-d81f18ff590f">